components/esp_modem/src/esp_modem_c_api.cpp

```diff
@@ -418,3 +418,8 @@ extern "C" esp_err_t esp_modem_command(esp_modem_dce_t *dce_wrap, const char *co
         }
     }, timeout_ms));
 }
+
+extern "C" esp_err_t esp_modem_set_baud(esp_modem_dce_t* dce_wrap, int baud)
+{
+	return command_response_to_esp_err(dce_wrap->dce->set_baud(baud));
+}
```

Comment:
you might need to install the pre-commit hook to help you with some formatting nitpicks, here I think, we miss a newline

Reply:
Thank you for your feedback! I've installed the pre-commit and let that run. Sorry for not properly reading the Contribution guidelines

Reply:
Thanks for the quick fix! Merging with squash function to keep the history clean.